modules/virt-editing-vm-yaml-web.adoc

```diff
@@ -1,14 +1,24 @@
 // Module included in the following assemblies:
 //
 // * virt/virtual_machines/virt-edit-vms.adoc
+//Establishing conditional expressions
+//
+ifeval::["{context}" == "virt-edit-vms"]
+:object: virtual machine
+:object-vm-details: Virtual Machine Details
+:object-vm-overview: Virtual Machine Overview
+endif::[]
+
+ifeval::["{context}" == "virt-editing-vm-template"]
+:object: virtual machine template
+:object-vm-details: Virtual Machine Template Details
+endif::[]
 
 [id="virt-editing-vm-yaml-web_{context}"]
 
 = Editing a virtual machine YAML configuration using the web console
 
-Using the web console, edit the YAML configuration of a virtual machine.
-
-Not all parameters can be updated. If you edit values that cannot be changed and click *Save*, an error message indicates the parameter that was not able to be updated.
+Using the web console, edit the YAML configuration of a virtual machine. Some parameters are not editable. If you edit values that cannot be changed and click *Save*, an error message indicates the parameter that was not able to be updated.
 
 The YAML configuration can be edited while the virtual machine is *Running*, however the changes will only take effect after the virtual machine has been stopped and started again.
```

Comment:
Passive voice.

Suggestion: If the virtual machine is running, changes will not take effect until you restart the virtual machine.

Reply:
Removed the sentence with the passive voice. Incorporated your suggestion by combining two sentences into one:

If you edit the YAML configuration while the virtual machine is running, changes will not take effect until you restart the virtual machine.